xarray/tests/test_conventions.py

```diff
@@ -129,9 +129,9 @@ def test_multidimensional_coordinates(self) -> None:
         foo1_coords = enc["foo1"].attrs.get("coordinates", "")
         foo2_coords = enc["foo2"].attrs.get("coordinates", "")
         foo3_coords = enc["foo3"].attrs.get("coordinates", "")
-        assert set(foo1_coords.split()) == {"lat1", "lon1"}
-        assert set(foo2_coords.split()) == {"lat2", "lon2"}
-        assert set(foo3_coords.split()) == {"lat3", "lon3"}
+        assert foo1_coords == "lon1 lat1"
```

Comment:
Why is this not `"lat1 lon1"` if it is sorted? (and same below e.g. `"time lon lat"`)

Reply:
User-specified coordinates attributes (or encodings further below) are not sorted, only internally assigned coordinates get sorted. User-specified attributes are already deterministic.